buildhub/main/management/commands/rebuild-bigquery.py

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import time
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from google.cloud import bigquery
+from google.cloud.exceptions import NotFound
+
+from buildhub.main.models import Build
+from buildhub.main.bigquery import get_schema_file_object
+
+logger = logging.getLogger("buildhub")
+
+
+class Command(BaseCommand):
+    help = "Create new table from Postgres into BigQuery via batched inserts."
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument(
+            "--yes",
+            action="store_true",
+            default=False,
+            help="Confirm overwriting tables without prompt",
+        )
+
+    def handle(self, *args, **options):
+        """See https://googleapis.dev/python/bigquery/latest/index.html."""
+
+        project_id = settings.BQ_PROJECT_ID
+        dataset_id = settings.BQ_DATASET_ID
+        table_id = f"{project_id}.{dataset_id}.{settings.BQ_TABLE_ID}"
+
+        client = bigquery.Client(project=project_id)
+        try:
+            client.get_table(table_id)
```

Comment:
This script would probably not be useful if a table already exists. There isn't a great mechanism for upserting (replacing rows) because BigQuery is append-only by nature.

Reply:
How much data are we talking about? Assuming we're talking data in the 10s/100s of megabytes, doing a compare of what's in buildhub vs. in bigquery then appending any rows missing from the latter seems like it might be reasonable? Yes it would take a while, but at least it would be automated

Reply:
We're looking at 1,727,065 rows * 16 bytes (per md5 hash), or about 30 mb to do the initial filtering. It would make this function a bit more complicated. I imagine the proper way to do this is to do something like:

```sql
WITH bigquery_hashes as (...),
SELECT *
FROM build
LEFT JOIN bigquery_hashes
USING (build_hash)
WHERE build_hash is NULL
```

and then iterating over the results of this query to insert into the table.  I'd prefer to leave as future work (i.e. out of scope of this PR), but this would be a good thing to have and schedule periodically.

Reply:
Leaving this for later sounds good to me.